Automattic/wp-calypso
# Theme Discovery: Launch Partner Collection on LoTS

Related to:
- https://github.com/Automattic/dotcom-forge/issues/4442

## Proposed Changes

* Changed config keys to launch the Partner collection on the LoTS

## Testing Instructions



* Checkout this branch locally.
* Run this command: `yarn feature-search themes/discovery-lots `
* You should see that the feature is enabled everywhere.
* You can also check the Calypso live link and it should be available.

## Pre-merge Checklist
